src/nrniv/bgpdma.cpp

```diff
@@ -724,13 +712,6 @@ static void bgpdma_cleanup() {
 	}}}
 }
 
-static void bgptimeout() {
-	printf("%d timeout %d %d %d\n", nrnmpi_myid, current_rbuf,
-		bgp_receive_buffer[current_rbuf]->nsend_,
-		bgp_receive_buffer[current_rbuf]->nrecv_
-	);
-}
-
 #if WORK_AROUND_RECORD_BUG
 static void ensure_ntarget_gt_3(BGP_DMASend* bs) {
```

Comment:
this is not used anywhere. what if we remove it?

Reply:
done